RedditSharp/Subreddit Information/SubredditImage.cs

```diff
@@ -35,18 +37,18 @@ public class SubredditImage
         public Uri Url { get; set; }
         public SubredditStyle SubredditStyle { get; set; }
 
-        public void Delete()
+        public async Task Delete()
```

Comment:
Create new async method instead of replacing old so we don't break people's code